feat: add formulation support and update documentation (#377)
MediaMarktSaturn/technolinator: docs/Deployment_Config.md

Btw: There's also a nice Helm chart for hosting Dependency-Track on Kubernetes available [here](https://github.com/MediaMarktSaturn/helm-charts/tree/main/charts/dependency-track).

> If Kubernetes isn't the right thing for you and you'd like something simpler, there's also a sample [docker compose file](/docs/docker-compose/docker-compose.sample.yml) and an accompanying [sample.env](/docs/docker-compose/sample.env) available.

Reviewer: ```suggestion
If Kubernetes doesn't fit your use case or you'd like something simpler, there's also a sample [docker compose file](/docs/docker-compose/docker-compose.sample.yml) and an accompanying [sample.env](/docs/docker-compose/sample.env) available.
```
